fiscal_epos_print_fiscalcode/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+#  Copyright 2020 Simone Rubino - Agile Business Group
+#  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'ITA - Includi il codice fiscale negli scontrini',
+    'version': '12.0.1.0.0',
+    'category': 'Point of Sale',
+    'summary': 'Allow to include fiscal code in fiscal receipt',
```

Comment:
Ciao, giusto una piccola nota.
Il nome in genere dovrebbe essere conciso. Quello attuale sembra più adatto per il `summary`, che spiega più dettagliatamente cosa fa il modulo.

Es.
`ITA - Codice fiscale (negli) scontrini` (vedi tu se usare la preposizione articolata o no)

Il summary poi andrebbe in italiano, come da linee guida. 
Es.
`Consente di includere/Include il codice fiscale negli scontrini`

Per il resto LGTM

Reply:
Grazie della review! Correggo subito.

Mi puoi dare il link delle linee guida? 
Così me lo segno e prossima volta magari ci azzecco :smile:  Solitamente guardo un altro modulo a caso del repository e copio :sweat_smile:

Reply:
Eccole! 😄 
https://www.odoo-italia.org/documentazione/12.0/sviluppo/guidelines.html

Reply:
> Eccole! 
> https://www.odoo-italia.org/documentazione/12.0/sviluppo/guidelines.html

Grazie, ma qui non si parla del summary.. Comunque l'ho cambiato

Reply:
> > Eccole!
> > https://www.odoo-italia.org/documentazione/12.0/sviluppo/guidelines.html
> 
> Grazie, ma qui non si parla del summary.. Comunque l'ho cambiato

Hai ragione. Probabilmente è una svista, vanno aggiornate.
La necessità di mettere name e summary in italiano per l10n-italy era nata dal fatto che attualmente in OCA/weblate non sono traducibili (a differenza dei moduli ODOO standard).

P.S.: a tale riguardo, ho appena chiesto a Martin come fanno a esportare automaticamente i campi e includerli nel modulo base di Transifex, vediamo cosa mi dice. :-)

Reply:
> > > Eccole!
> > > https://www.odoo-italia.org/documentazione/12.0/sviluppo/guidelines.html
> > 
> > 
> > Grazie, ma qui non si parla del summary.. Comunque l'ho cambiato
> 
> Hai ragione. Probabilmente è una svista, vanno aggiornate.

Fatto in https://github.com/odoo-italia/documentazione/commit/68d7db55016deaf7a39277411471c7d172e8a776